src/app/job-status/job-status.component.html

```diff
@@ -19,6 +19,7 @@
       <div class="job-info__progress-bar__wrapper">
         <div class="job-info__progress-bar" [style.width]="job.progress + '%'"></div>
       </div>
+      <div class="job-info__progress-bar-percentage">{{ job.progress }}<span style="font-size: 75%">%</span></div>
```

Comment:
I'll have a look whether we can somehow work around the third percentage element (might take a little while).